modules/nw-ipfailover-configuration.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ipfailover.adoc
+
+[id="nw-ipfailover-configuration_{context}"]
+= Configuring IP failover
+
+As a cluster administrator, you can configure IP failover on an entire cluster, or on a subset of nodes, as defined by the label selector. You can also configure multiple IP failover deployment configurations in your cluster, where each one is independent of the others.
+
+The IP failover deployment configuration ensures that a failover pod runs on each of the nodes matching the constraints or the label used.
+
+This pod runs Keepalived which uses Virtual Router Redundancy Protocol (VRRP) among all the Keepalived daemons to ensure that the service on the watched port is available, and if it is not, Keepalived automatically floats the virtual IPs (VIP).
+
+For production use, make sure to set a `selector` with at least two nodes to select the nodes. Also, set a `replicas` value that matches the number of nodes for the given labeled `selector`.
+
+.Prerequisites
+
+* You are logged in to the cluster with a user with `cluster-admin` privileges.
+
+.Procedure
+
```

Comment:
Before creating `Deployment`, seems need to create serviceaccount "ipfailover" and update SCC. e.g.
```
$ oc create sa ipfailover
$ oc adm policy add-scc-to-user privileged -z ipfailover
```
@rfredette Could you help confirm?

Reply:
Yes, that's correct. I believe it also needs the `hostnetwork` SCC, so this should be what is needed:
```
$ oc create sa ipfailover
$ oc adm policy add-scc-to-user privileged -z ipfailover
$ oc adm policy add-scc-to-user hostnetwork -z ipfailover
```

Reply:
Added